libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/building/api/MapboxBuildingsApi.kt

```diff
@@ -43,10 +44,8 @@ class MapboxBuildingsApi internal constructor(
      * - A point you can obtain that lives on a building also known as location point
      * - A point you would route to on the road also known as routable point
      *
-     * For this method to work, your route's original request should be a "location point" and not a
-     * "routing point", otherwise, nothing will be returned. If the coordinate is a routing point
-     * (which is the correct way to do it), you should define a "waypoint_targets" in the original
-     * request, for each of the waypoints, and for the final destination.
+     * For this method to work, the [point] passed to this API should be a "location point" and not a
+     * "routing point".
```

Comment:
```suggestion
     * For this method to work, the [point] passed to this API should be a "location point" (where the actual building is)  and not a
     * "routing point" (point on a road snapped from the address).
```

Reply:
Same everywhere below, let's try to clarify what routable and location points are in this context as this might be new to a lot of developers using this API.

Reply:
I have already defined both the points just above it. Although changed it a little bit based on your suggestion.